development/fixing-e2e-tests.md

```diff
@@ -0,0 +1,28 @@
+# How to fix e2e tests
+
+End to end (e2e) tests can be really difficult to debug - sometimes they fail seemingly at random, and sometimes they only fail on certain environments (eg: ci but not locally). This can make reproducing and reliably fixing the issue challenging, so here are some tips to help!
+
+## Read the protractor logs
+
+Read the failure carefully - it often has really good info but sometimes it's just hard to find. Most importantly it tells you exactly the line in the test that failed and you can look that up in the source to see what protractor was trying to do. The error message itself is also really useful. Also sometimes one error causes the next, so always start with the first test failure before looking at the others.
```

Comment:
I think the second part of this is super helpful and my warrant being split off into its own bullet point.